casper/src/main/scala/coop/rchain/casper/engine/Initializing.scala

```diff
@@ -140,7 +141,11 @@ class Initializing[F[_]
       // Request all blocks for Last Finalized State
       blockRequestStream <- LastFinalizedStateBlockRequester.stream(
                              approvedBlock,
-                             blockMessageQueue
+                             blockMessageQueue,
+                             CommUtil[F].broadcastRequestForBlock,
+                             BlockStore[F].contains,
```

Comment:
@tgrospic I'm curious if it makes sense to put these as implicits?

So instead of e.g. 
```scala
stream[F[_]: Concurrent: Time: BlockStore: CommUtil: Log]
``` we can have

```scala
stream[F[_]: Concurrent: Time:  StoreBlock: BroadcastRequestForBlock: Log]
```
and have a large database of all operations that node can do via all aggregate instances implementing these `operations` traits